cloudevents/sdk/converters/util.py

```diff
@@ -0,0 +1,10 @@
+import typing
+
+
+def has_binary_headers(headers: typing.Dict[str, str]) -> bool:
+    return (
+        "ce-specversion" in headers
+        and "ce-source" in headers
+        and "ce-type" in headers
+        and "ce-id" in headers
+    )
```

Comment:
Do we really need a separate file? Can we put this method in an existing file?

Reply:
I wanted both [BinaryHTTPCloudEventConverter](https://github.com/cloudevents/sdk-python/blob/master/cloudevents/sdk/converters/binary.py) and [JSONHTTPCloudEventConverter](https://github.com/cloudevents/sdk-python/blob/master/cloudevents/sdk/converters/structured.py) to have access to this method. I could move both files into one file and move that function into that file. That would just be up to preferences I think.

Reply:
Sounds good.